src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -156,11 +200,31 @@ NavHeader.propTypes = {
   setMemberCount: PropTypes.func,
   memberCount: PropTypes.number,
   topicDetail: PropTypes.object,
-  setIsFollow: PropTypes.func
+  setIsFollow: PropTypes.func,
+
+  searchText: PropTypes.string,
+  setSearchText: PropTypes.func,
+  setDiscoveryLoadingData: PropTypes.func,
+  isFocus: PropTypes.bool,
+  setIsFocus: PropTypes.func,
+  fetchDiscoveryData: PropTypes.func,
+  onCancelToken: PropTypes.func,
+  placeholderText: PropTypes.string,
+  setIsFirstTimeOpen: PropTypes.func,
+  getSearchLayout: PropTypes.func,
+  hasSearch: PropTypes.bool
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New prop types have been added. Ensure that these props are being passed in correctly where this component is used, and that they have appropriate default values if they are optional.
```diff
+   searchText: PropTypes.string,
+   setSearchText: PropTypes.func,
+   setDiscoveryLoadingData: PropTypes.func,
+   isFocus: PropTypes.bool,
+   setIsFocus: PropTypes.func,
+   fetchDiscoveryData: PropTypes.func,
+   onCancelToken: PropTypes.func,
+   placeholderText: PropTypes.string,
+   setIsFirstTimeOpen: PropTypes.func,
+   getSearchLayout: PropTypes.func,
+   hasSearch: PropTypes.bool
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->